### PR TITLE
MS-875 Language change fixes

### PR DESCRIPTION
**Pull request by luhmirin-s**
* Cherry-picked the fix for consent text resolving using activity context.
* Fixed a similar issue with age range names.

### CODE REVIEW

**Comment by luhmirin-s** [comment]
@meladRaouf Can the duplication check be overriden? I believe it is a false positive in this case.

File: feature/select-subject-age-group/src/main/java/com/simprints/feature/selectagegroup/screen/BuildAgeGroupsUseCase.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.simprints.infra.config.store.ConfigRepository
+import com.simprints.infra.config.store.models.AgeGroup
+import com.simprints.infra.config.store.models.sortedUniqueAgeGroups
+import javax.inject.Inject
+
+internal class BuildAgeGroupsUseCase @Inject constructor(
+    private val configurationRepo: ConfigRepository,
+) {
+
+    /**
```
Comment (meladRaouf) [inline feature/select-subject-age-group/src/main/java/com/simprints/feature/selectagegroup/screen/BuildAgeGroupsUseCase.kt:12]: Is this code comment still valid?

**Review by meladRaouf — APPROVED** [review]

**Comment by meladRaouf** [comment]
> @meladRaouf Can the duplication check be overriden? I believe it is a false positive in this case.

We can’t override this validation, but we can still merge the PR.

File: feature/select-subject-age-group/src/main/java/com/simprints/feature/selectagegroup/screen/BuildAgeGroupsUseCase.kt (hunk 1 above)
Comment (luhmirin-s) [reply feature/select-subject-age-group/src/main/java/com/simprints/feature/selectagegroup/screen/BuildAgeGroupsUseCase.kt:12]: It is still valid, I just removed the part that was referring to display text.

**Pull request merged**